src/test/java/seedu/address/model/person/passenger/NameContainsKeywordsPredicateTest.java

```diff
@@ -47,20 +50,23 @@ public void equals() {
     public void test_nameContainsKeywords_returnsTrue() {
         // One keyword
         NameContainsKeywordsPredicate predicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList(VALID_NAME_AMY.split("\\s+")[0]));
+                new NameContainsKeywordsPredicate(Collections.singletonList(
+                        VALID_NAME_AMY_LOWER_CASE.split("\\s+")[0]));
         assertTrue(predicate.test(new PassengerBuilder().withName(VALID_NAME_AMY).build()));
 
         // Multiple keywords
-        predicate = new NameContainsKeywordsPredicate(Arrays.asList(VALID_NAME_AMY.split("\\s")));
+        predicate = new NameContainsKeywordsPredicate(Arrays.asList(VALID_NAME_AMY_LOWER_CASE.split("\\s")));
         assertTrue(predicate.test(new PassengerBuilder().withName(VALID_NAME_AMY).build()));
 
         // Only one matching keyword
-        predicate = new NameContainsKeywordsPredicate(Arrays.asList(VALID_NAME_AMY.split("\\s")[0],
+        predicate = new NameContainsKeywordsPredicate(Arrays.asList(VALID_NAME_AMY_LOWER_CASE.split("\\s")[0],
                 VALID_NAME_BOB.split("\\s")[0]));
         assertTrue(predicate.test(new PassengerBuilder().withName(VALID_NAME_AMY).build()));
 
         // Mixed-case keywords
-        predicate = new NameContainsKeywordsPredicate(Arrays.asList("aMy", "beE"));
+        // Keywords changed to lowercase as parser will change all keywords to lowercase
```

Comment:
Maybe add this comment to the earlier testcases also? might make it more clear why everything has been made to lowercase.